backend/services/feed.ts

```diff
@@ -108,6 +47,23 @@ async function getPostContent(msg: AnnouncementResponse): Promise<[number, Post]
       responseType: 'text',
       timeout: 10000,
     });
+    const replyMessages = ContentRepository.get({ announcementTypes: [AnnouncementType.Reply] });
+    const replies: Reply[] = await Promise.all(
+      replyMessages.map(async (replyMessage): Promise<Reply> => {
+        const replyAnnouncement = replyMessage.announcement as ReplyAnnouncement;
+        const replyResp = await axios.get(translateContentUrl(replyAnnouncement.url), {
+          responseType: 'text',
+          timeout: 10000,
+        });
+
+        return {
+          fromId: replyAnnouncement.fromId,
+          contentHash: replyAnnouncement.contentHash,
+          content: replyResp.data as string,
+          timestamp: new Date().toISOString(), // TODO: use block timestamp
```

Comment:
shouldn't this have the value `inReplyTo`?

Reply:
I believe `inReplyTo` is inside the `content` JSON.

Reply:
misread. thought we were returning reply content, not post content. looks great!